one-point-oh.md

```diff
@@ -0,0 +1,146 @@
+# 1.0 plan and priorities
+
+We don't have time for big changes or big features (dev time + time to marinate). We know vaguely where we're going (see other design PRs).
+
+High level priorities:
+
+- Stability and polish
+  - bugs, performance, glitches, etc.
+- User experience
+  - Goal: KCL is easy to read and comprehend
+  - Feedback is useful (though perhaps not pretty)
+    - IDE experience
+    - Error messages (parser in particular)
+- Supporting core new features in engine, as required
+- Future compatibility
+- Not making tech debt worse
+  - And ideally a little bit better
+
+Related:
+
+- [Modelling app roadmap](https://github.com/orgs/KittyCAD/projects/35)
+- [1.0 milestone](https://github.com/KittyCAD/modeling-app/milestone/2)
+- [CAD primitives roadmap](https://github.com/KittyCAD/modeling-app/issues/729)
+
+Priorities in **bold**, dependencies in *italics*, task size in t-shirt sizes (taking into account breaking change management; m-l means uncertainty about which, not 'somewhere in between').
+
+## As required
+
+### Bug fixing
+
+**P0**:
+
+- high priority bugs as they emerge
+- plan to seek out and address lower-priority bugs during feature freeze
+
+
+### Non-language feature work
+
+Priority follows from other team's priorities.
+
+New or more flexible std lib functions and code mods are expected as side effects of other work.
+
+New features which might require significant work:
+
+- [Multiple profiles in Sketches](https://github.com/KittyCAD/modeling-app/issues/1876)
+- Assemblies?
+
+
+### Pre-paying technical debt
+
+**P1**:
+
+- Don't make things worse by adding hacks, especially around the API of KCL
+
+**P3**:
+
+- Supplying user-side IDs to engine
+- code mod API
+
+
+## Language priorities
+
+**P1** (priority ordered):
+
+- reserve keywords (s)
+  - https://github.com/KittyCAD/modeling-app/issues/4486
+  - and give meaning to underscore in identifiers
+- function args
+  - named and optional args (m; *$0*)
+  - only one non-named arg (m; *$1*)
+    - ideally we make it the first arg, but doesn't have to be
+    - could require it be called `self`, but I don't think we need to and that makes back-compat harder
+- simple tidying changes
+  - automatic conversion of round floats to ints, remove the `int()` function (s)
+  - record init: `:` -> `=`  (s)
+  - function decl: remove `=` and `->`, `:` for type annotations (s)
+    - return type syntax?
+  - convert special strings (e.g., `END`, `XY`) to constants (s)
```

Comment:
We could add these to the prelude where `ZERO` and `HALF_TURN` are defined. As far as I know, the stdlib doesn't have any constant values, only functions.